README.md

```diff
@@ -139,16 +139,18 @@ And then we just close off the file.
 
 ### Configuring the control system
 
-The configuration file for the control system depends on the chosed control
+The configuration file for the control system depends on the chosen control
 system adapter. If no file is provided, it will default to expose everything to
 the control system.
 
 #### DOOCS
 
-The file for DOOCS has to be named
-`generic_chimeratk_server-DoocsVariableConfig.xml`. It might also be advisable,
-though not necessary, to provide a DOOCS server configuration file containing
-permissions and such. For details, refer to the [documentation of the DOOCS
+DOOCS requires a basic server configuration file named `doocs-generic-chimeratk-server01.conf` to set `SVR.RPC_NUMBER` 
```

Comment:
The question is how much information we want to give about configuring a DOOCS server at all. A minimal version to get a server running at all looks like this:
```
eq_conf: 

eq_fct_name: 	"LOCAHOST._SVR"
eq_fct_type: 	1
{
SVR.RPC_NUMBER:  	610498009
}
```
However, it is not really usable in production context. The minimum you want is to set SVR.RATE, SVR.STORE.RATE, and SVR.STORE.AUTO. These are needed to have the input data persisted (I am a bit unsure which ones are really needed, but I think if SVR.RATE is not set, update is never called and the config file is never written.

We should clean up the config file to the minimum reasonable. It looks something like this (taken from piezo server):
```
eq_conf: 

oper_uid:   -1
oper_gid:   ${OPERATOR_GID}
xpert_uid:  0
xpert_gid:  0

eq_fct_name:    "${SVR_LOCATION}"
eq_fct_type:    1
{
SVR.RATE:          1  0  0  0
SVR.PROGRAMMER:    "M. Killenberg"
SVR.STORE.RATE:    60
SVR.STORE.AUTO:    10
SVR.HOST_NAME:     ${HOSTNAME}
SVR.RPC_NUMBER:    ${rpcNumber}
SVR.WDADDR:        "${WATCHDOG_ADDRESS}"
SVR.FACILITY:      "${FACILITY}"
SVR.DEVICE:        "${DEVICE}"
}
```

Reply:
You only specify the server location here. The other locations are added at run time by the generic server.

Reply:
I have removed the whole doocs section and just referred to the DOOCS CSA docs